App.js

```diff
@@ -1,9 +1,17 @@
 import * as Font from 'expo-font';
 import * as SplashScreen from 'expo-splash-screen';
 import React, { useEffect, useState } from 'react';
+import * as Sentry from '@sentry/react-native';
 
 import { MainApp } from './src';
-import { fontConfig } from './src/config';
+import { fontConfig, sentryApi } from './src/config';
+
+Sentry.init({
+  dsn: sentryApi.dsn,
+  enableNative: false, // Note: Native crash reporting is not available with the classic build system (expo build:[ios|android]), but is available via EAS Build.
+  enableInExpoDevelopment: true,
```

Comment:
Hey @donni106, I guess we want to turn this off as all errors while development will go to sentry if we leave this on.

Reply:
true. i will comment it out, so that devs can quickly enable it if necessary.

Reply:
done in e3046032453d60f92be801e71f824b42c134df45